services/py-api/src/database/repository/teams_repository.py

```diff
@@ -29,6 +29,9 @@ async def create(
         **kwargs: Dict[str, Any]
     ) -> Result[Team, DuplicateTeamNameError | Exception]:
 
+        if input_data.team_name is None:
```

Comment:
If you're using types, how come `team_name` can be null?

Reply:
it was because of mypy. I decided to do it like that, but we could also put a comment to ignore mypy err. Look at the input data schema and the Team obj model

Reply:
I'm not sure why you would be accepting something which is incorrect in the first place. Is this only for the sake of reusing the Team model?

Reply:
Hey Misho! We are going to keep this in mind for later on. For now we should advance in a slightly faster pace